UPCommons.js

```diff
@@ -96,8 +139,13 @@ var testCases = [
 						"creatorType": "author"
 					},
 					{
-						"firstName": "Miguel Angel",
-						"lastName": "Ahumada Sempoal",
+						"firstName": "Ahumada",
```

Comment:
how confident are you that we can fix the author issue in EM? Seems a bit tricky because we do also find multiple creators in a single DC tag. If we think it might be hard to fix in EM, we'll probably want to parse authors here separately. Your call -- I don't mind taking this while we're working on EM.